testprojects/pants-plugins/tests/pants_test/new_package/BUILD

```diff
@@ -0,0 +1,7 @@
+python_tests(
+  dependencies=[
+    'tests/python/pants_test:test_infra',
+    'testprojects/pants-plugins/3rdparty/pants',
+  ],
+  tags={'integration'},
```

Comment:
This isn't currently an integration test, but maybe it should be? See below.

Reply:
Thanks for pointing this out here too -- this was removed as well.